CMakeLists.txt

```diff
@@ -60,6 +60,10 @@ cmake_dependent_option(USE_SSE "Build with x86 SSE instruction support" ON
   "CMAKE_SYSTEM_PROCESSOR STREQUAL x86_64 OR CMAKE_SYSTEM_PROCESSOR STREQUAL amd64" OFF)
 option(USE_F16C "Build with x86 F16C instruction support" ON) # autodetects support if ON
 option(USE_LAPACK "Build with lapack support" ON)
+option(USE_ILP64_LAPACKE "Use ILP64 LAPACKE interface" OFF)
+option(USE_LAPACKE_INTERFACE "Use LAPACKE interface for lapack support" OFF)
+option(LINK_GFORTRAN "Link to gfortran" OFF)
```

Comment:
gfortran is a transitive dependency of gomp. This should be handled automatically

Reply:
This flag is used on Centos. On Centos I tried a omp=1 openblas build and I think gomp clashed with mxnet llvm omp and one of the testcase would just hang until time-out. Building openblas with omp=0 resolved the issue. So there since we are not linking gomp we sort of need this flag.

Reply:
If we don't link gomp we shouldn't need gfortran. It suggests that something is wrong here. Are you referring to the CD build or CI build?

Reply:
Ok, nevermind. You are probably building openblas without omp but with the fortran based functions. I mixed those up.

Reply:
We should be able to use https://cmake.org/cmake/help/latest/module/CheckSymbolExists.html or https://cmake.org/cmake/help/latest/module/CheckCSourceCompiles.html#module:CheckCSourceCompiles to detect this